app/models/katello/docker_manifest_platform.rb

```diff
@@ -0,0 +1,9 @@
+module Katello
+  class DockerManifestPlatform < Katello::Model
```

Comment:
Maybe this should be a module mixed into `DockerManifestListManifestPlatform` and `DockerManifestManifestPlatform`? Or, perhaps it could be the superclass for those two classes. *Or*, maybe we should use single table inheritance here. It just seems like there's some similar functionality here that we could DRY up and organize.

Reply:
@jlsherrill - Can you review and reply on these models? If you recall, I had to work quite extensively with you to get this these models and relations correct.